pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "apps"
+version = "0.2.1"
+description = "Django template package"
+license = {file = "LICENSE"}
+requires-python = ">=3.9"
+optional-dependencies = {front = ["npm"]}
```

Comment:
but npm is not a python package. Is it really usefull to put this here?

Reply:
Not necessary.
Technically It's recognized as real nodejs dependency.
https://peps.python.org/pep-0621/#other-names-for-dependencies-optional-dependencies
That pep describe some possibility of no python package keyword can be required.